### PR TITLE
Naming the files according to the rfc 'shortname'

### PR DESCRIPTION
**Pull request by piogor**
An alternative to #71

### CODE REVIEW

**Comment by lorenzopivetta** [comment]
Just one more comment... should we consider using leading "0" (zero), e.g. 01/Tango.md, 02/Device.md... for a proper alphanumeric sorting?

**Comment by hardion** [comment]
I prefer UPPERCASE for the file name i.e DEVICE.md

**Comment by lorenzopivetta** [comment]
> I prefer UPPERCASE for the file name i.e DEVICE.md

Is there a specific reason for?

**Comment by hardion** [comment]
You can forget my comment. I thought the Shortname was written in uppercase in the md file.

File: README.md [hunk 1]
```diff
@@ -14,22 +14,22 @@ Read more [here](https://github.com/tango-controls/rfc/wiki).
...
+[RFC-6](6/Database.md)   | The database system                                           | Draft      | Gwenaelle Abeillé
+RFC-7                    | The pipe model                                                | Raw        | Reynald Bourtembourg
+RFC-8  | The server model                                               | Raw          | Lorenzo Pivetta
+RFC-9  | Data types                                                     | Raw          | Gwenaelle Abeillé
 RFC-10 | The Request-Reply protocol                                     | Raw          | Reynald Bourtembourg
 RFC-11 | The Request-Reply protocol - CORBA implementation              | Raw          | 
 RFC-12 | The Publisher-Subscriber protocol                              | Raw          | Vincent Hardion 
 RFC-13 | The Publisher-Subscriber protocol - ZeroMQ implementation      | Raw          | 
-RFC-14 | Logging service                                                | Raw          | Sergi Blanchi-Torné
+RFC-14(14/Logging.md) | Logging service                                                | Raw          | Sergi Blanchi-Torné
```
Comment (mliszcz) [inline README.md]: Should be `[RFC-14](14/Logging.md)`

**Review by mliszcz — APPROVED** [review]
Looks good except that one link seems to be broken

File: README.md (hunk 1 above)
Comment (piogor) [reply README.md]: Thanks, fixed with the recent commit.

**Pull request merged**